src/app/home/home.component.html

```diff
@@ -408,7 +408,7 @@ <h5 class="card-title">
                 </div>
             </div>
             <ul class="list-group list-group-flush listStyling">
-                <div id="spinnerBorder" class="spinner-border" role="status">
+                <div class="spinner-border" role="status">
```

Comment:
Any idea if class 'spinner-border' was used anywhere @Kledis98 ?

Reply:
I dropped the ID and utilized the class for the styling.

Reply:
It is all right. It was used only there, indicating the loading state.